contracts/plugins/assets/morpho-aave/MorphoTokenisedDeposit.sol

```diff
@@ -15,11 +15,25 @@ struct MorphoTokenisedDepositConfig {
 }
 
 abstract contract MorphoTokenisedDeposit is RewardableERC4626Vault {
+    uint256 private constant PAYOUT_PERIOD = 7200;
```

Comment:
this is only 2 hours. should probably be at least a day, maybe a week?

Reply:
It's 7200 blocks, so it will be paid out over 24 hrs on mainnet.